nvm.sh

```diff
@@ -146,7 +146,12 @@ nvm_has_system_iojs() {
 }
 
 nvm_is_version_installed() {
-  [ -n "${1-}" ] && [ -x "$(nvm_version_path "$1" 2>/dev/null)"/bin/node ]
+  local NVM_OS
+  NVM_OS="$(nvm_get_os)"
+  local NODE
+  NODE='node'
+  [ "_${NVM_OS}" = "_win" ] && NODE='node.exe'
+  [ -n "${1-}" ] && [ -x "$(nvm_version_path "$1" 2>/dev/null)"/bin/"${NODE}" ]
```

Comment:
let’s restructure this to if/else, so the `-n` is checked before calling nvm_get_os.

also, which is cheaper: calling nvm_get_os, or doing an extra -x check?

Reply:
I have changed it a bit and still call nvm_get_os, but I was thinking that maybe we could just check the existence of node.exe even if system if not win, so it avoids calling nvm_get_os.
Would it be not better ?

Reply:
i kind of prefer the explicitness of only checking `.exe` on windows, and only non-exe on non-windows.